packages/vuetify/src/components/VDatePicker/__tests__/VDatePicker.date.spec.ts

```diff
@@ -26,7 +26,15 @@ describe('VDatePicker.ts', () => { // eslint-disable-line max-statements
         ...options,
         mocks: {
           $vuetify: {
-            lang: new Lang(),
+            lang: new Lang({
+              locales: {
+                en: {
+                  datePicker: {
+                    itemsSelected: 'i has {0} items',
```

Comment:
is it just testing? why not adding thuis to locale

Reply:
Yeah, it's for testing, I don't get what you mean